.mypy.ini

```diff
@@ -31,35 +28,56 @@ mypy_path = executorch
 [mypy-executorch.codegen.*]
 follow_untyped_imports = True
 
-[mypy-executorch.extension.*]
+[mypy-executorch.devtools.*]
 follow_untyped_imports = True
 
 [mypy-executorch.exir.*]
 follow_untyped_imports = True
 
+[mypy-executorch.extension.*]
+follow_untyped_imports = True
+
 [mypy-executorch.kernels.*]
 follow_untyped_imports = True
 
+[mypy-executorch.profiler.*]
+follow_untyped_imports = True
+
 [mypy-executorch.runtime.*]
 follow_untyped_imports = True
 
+[mypy-requests.*]
+follow_untyped_imports = True
+
 [mypy-torchgen.*]
 follow_untyped_imports = True
 
-[mypy-setuptools.*]
+[mypy-buck_util]
 ignore_missing_imports = True
 
-[mypy-buck_util]
+[mypy-docutils.*]
```

Comment:
why these one has ignore missing imports=true? wasnt before? Same for pandas

Reply:
I am guessing to make linter pass?

Reply:
i just sorted alphabetically 

and some third party libraries, it does the best effort to infer the types, like, `follow_untyped_imports=True`

if it can't traverse due to missing symbols, linters complains, so i'm just ignoring these imports altogether. ignore_missing_imports